README.md

```diff
@@ -1,4 +1,4 @@
-Highlight TODO and similar keywords in comments and strings
+Highlight TODO and similar keywords (codetags) in comments and strings
```

Comment:
Lets stick to the current terminology. Maybe codetags would have been the better term but then it should be introduced in a sentence not unexplained in a heading.

Lets not do this at all in this commit, because it is unrelated to the purpose of this commit. If you still want to do this as well, then do it in a second commit.